src/App.tsx

```diff
@@ -1,33 +1,32 @@
-import React, {useEffect, useState} from 'react';
-import './App.css';
-import {Player, Home, CreateVideo} from './Components/';
-
-
-import sampimage1 from './Media/image1.png';
-// import samplevideo1 from './Media/video2.mp4';
-
-import ReactPlayer, { SourceProps } from 'react-player/lazy';
-
-import { BrowserRouter as Router, Switch, Route} from 'react-router-dom'; 
+import React, { useState } from "react";
+import "./App.css";
+import { ImportModal, Header } from "./Components/";
```

Comment:
We prefer to import components separately, rather than using an index file
So:
```
import { Header } from './Components/Header/Header';
import { ImportModal } from './Components/ImportModal/ImportModal';
```
Also prefer to keep all these imports in alphabetical order, with relative imports at the bottom